.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: php
 
 php:
+  - 7.1
```

Comment:
I had kept 7.0 first so that coverage builds run first and report are send to codecov earlier :)